engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register.html

```diff
@@ -15,7 +15,7 @@ <h2 translate>Content Host Registration</h2>
       </li>
       <li>
         <p translate>Install the pre-built bootstrap RPM:</p>
-        <pre><code>rpm -Uvh https://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
+        <pre><code>curl --insecure --output katello-ca-consumer.rpm https://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}<br>yum localinstall katello-ca-consumer.rpm</code></pre>
```

Comment:
I think you should use `{{ consumerCertRPM }}` everywhere you have `katello-ca-consumer.rpm` because it's [configurable](https://github.com/Katello/katello/blob/865101a111b8a94db1b13cec8a10f4b0046ef8f8/engines/bastion_katello/lib/bastion_katello/engine.rb#L11).